src/Downloader.php

```diff
@@ -68,6 +68,8 @@ public function allowContrib(bool $allow): void
      *
      * @param string $path    The path to get on the server
      * @param array  $headers An array of HTTP headers
+     *
+     * @return Response
```

Comment:
IDEs & cs-fixers consider that if you set a "partial" phpdoc, it's inconsistent. But yes, we shouldn't consider IDEs as almighty 🙂 I'm gonna remove useless doc